auditlog/receivers.py

```diff
@@ -61,6 +61,21 @@ def log_delete(sender, instance, **kwargs):
         )
 
 
+def log_access(sender, instance, **kwargs):
+    """
+    Signal receiver that creates a log entry when a model instance is accessed in a AccessLogDetailView.
+
+    Direct use is discouraged, connect your model through :py:func:`auditlog.registry.register` instead.
+    """
+    if instance.pk is not None:
+
+        LogEntry.objects.log_create(
+            instance,
+            action=LogEntry.Action.ACCESS,
+            changes="null",
```

Comment:
Hi @alieh-rymasheuski, if you take a look at the delete variant of the receiver, you'll see that changes is the result of
``` python
changes = model_instance_diff(instance, None)
```
see https://github.com/jazzband/django-auditlog/blob/master/auditlog/receivers.py#L55

In the context of access, this will always be None, as we pointed out above, see
https://github.com/jazzband/django-auditlog/pull/436#pullrequestreview-1137603666

The encoding into json will therefore return `'null'`, so we can skip all these calls and just put in `'null'` directly.
```
In [1]: import json

In [2]: json.dumps(None)
Out[2]: 'null'
```